tests/e2e/lib/backup.go

```diff
@@ -24,17 +31,24 @@ func CreateBackupForNamespaces(ocClient client.Client, veleroNamespace, backupNa
 			SnapshotMoveData:         &snapshotMoveData,
 		},
 	}
-	err := ocClient.Create(context.Background(), &backup)
-	return backup, err
+	return ocClient.Create(context.Background(), &backup)
```

Comment:
Curious, Why did we do this refactor ?

Reply:
I was thinking about https://github.com/openshift/oadp-operator/issues/1240 to make these functions more simple (but did not change nothing on them, except for not returning a backup and not having backup as a parameter anymore)